use bot::error::Error;

fn main() -> Result<(), Error> {
    Ok(())
}
